assets/src/edit-story/components/popup/index.js

```diff
@@ -67,7 +67,8 @@ const Container = styled.div.attrs(
 )`
   max-height: ${({ maxHeight }) =>
     maxHeight > 0 ? maxHeight + 'px' : 'initial'};
-  overflow-y: auto;
+  ${({ showOverflow }) =>
+    showOverflow ? 'overflow: visible;' : 'overflow-y: auto;'}
```

Comment:
Q: Why is the visible overflow needed -- is this something that would get fixed properly by #1931 ?